### PR TITLE
mcs: add service role

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

This PR changed the original role to table format and made other services' primary can be shown.

```commit-message
```

### Check List

Tests

- Manual test 

New template in microservice mode:
Screenshot 2023-10-08 at 16 25 12

New template in PD mode:
Screenshot 2023-10-08 at 16 23 39

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: pkg/mcs/resourcemanager/server/server.go [hunk 1]
```diff
@@ -382,8 +386,8 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	// Flushing any buffered log entries
 	defer log.Sync()
 
-	versioninfo.Log("Resource Manager")
-	log.Info("Resource Manager config", zap.Reflect("config", cfg))
+	versioninfo.Log(serviceName)
+	log.Info("Resource manager config", zap.Reflect("config", cfg))
```
Comment (lhy1024) [inline pkg/mcs/resourcemanager/server/server.go:390]:
```suggestion
	log.Info("resource manager config", zap.Reflect("config", cfg))
```

File: pkg/mcs/scheduling/server/server.go [hunk 2]
```diff
@@ -533,8 +539,8 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	// Flushing any buffered log entries
 	defer log.Sync()
 
-	versioninfo.Log("Scheduling")
-	log.Info("Scheduling config", zap.Reflect("config", cfg))
+	versioninfo.Log(serviceName)
+	log.Info("Scheduling service config", zap.Reflect("config", cfg))
```
Comment (lhy1024) [inline pkg/mcs/scheduling/server/server.go:543]: ditto

File: pkg/mcs/tso/server/server.go [hunk 3]
```diff
@@ -450,8 +452,8 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	// Flushing any buffered log entries
 	defer log.Sync()
 
-	versioninfo.Log("TSO")
-	log.Info("TSO config", zap.Reflect("config", cfg))
+	versioninfo.Log(serviceName)
+	log.Info("TSO service config", zap.Reflect("config", cfg))
```
Comment (lhy1024) [inline pkg/mcs/tso/server/server.go:456]: ditto

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**